Spigot-API-Patches/0283-Can-Fly-Fall-Damage.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TreyRuffy <TreyRuffy@users.noreply.github.com>
+Date: Sat, 13 Mar 2021 13:55:27 -0700
+Subject: [PATCH] Can Fly Fall Damage
+
+
+diff --git a/src/main/java/org/bukkit/entity/HumanEntity.java b/src/main/java/org/bukkit/entity/HumanEntity.java
+index f0e0710fef5a3e0b722ece7ccf89c3d0f88f8f0f..70b2617dc9ef4cdaa04ad499f7d3211c50e42315 100644
+--- a/src/main/java/org/bukkit/entity/HumanEntity.java
++++ b/src/main/java/org/bukkit/entity/HumanEntity.java
+@@ -576,6 +576,25 @@ public interface HumanEntity extends LivingEntity, AnimalTamer, InventoryHolder
```

Comment:
This seems tied to Player#getAllowFlight(). I can't think of any api reasons to implement it on HumanEntity. It would be my opinion to move it to the Player interface.

Reply:
Alright I have done this